src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -137,7 +137,6 @@ public static void DigestAuthentication_RoundTrips_Echo()
 
     [Fact]
     [OuterLoop]
-    [ActiveIssue(604, PlatformID.AnyUnix)]
     public static void NtlmAuthentication_RoundTrips_Echo()
```

Comment:
Apologies for the delayed response. libcurl and hence HttpClient on Unix support NTLM with explicit credentials. However default credentials are a no-op on Unix. @kapilash did propose a way to implement this using .netrc files but there is currently no planned work for this.